docs/src/developers_guide/contributing_pytest_conversions.rst

```diff
@@ -73,4 +74,6 @@ Common Translations
      - ``with pytest.raises(cls, match=msg_re):``
    * - ``mock.patch(...)``
      - ``mocker.patch(...)``
+   * - ``with mocker.patch.object(...) as x:``
```

Comment:
```suggestion
   * - ``with mock.patch.object(...) as x:``
```

Reply:
I _think_ that either syntax is OK, and that it would be less confusing to stick with `mocker` everywhere.